docs/reference/commandline/cli.md

```diff
@@ -1,408 +0,0 @@
----
-title: "Use the Docker command line"
-description: "Docker's CLI command description and usage"
-keywords: "Docker, Docker documentation, CLI, command line, config.json, CLI configuration file"
-aliases:
-  - /reference/commandline/cli/
-  - /engine/reference/commandline/engine/
-  - /engine/reference/commandline/engine_activate/
-  - /engine/reference/commandline/engine_check/
-  - /engine/reference/commandline/engine_update/
```

Comment:
There's a bunch of redirects here; looks like we may need to move those to a different page now

Reply:
Yep, they are in content/reference/cli/docker/_index.md @ docker/docs#19794